### PR TITLE
feat: interchain station docs

### PR DESCRIPTION
**Pull request by ericHgorski**

### CODE REVIEW

File: docs/develop/terrain/mint-an-nft.mdx [hunk 1]
```diff
@@ -284,7 +284,7 @@ Congratulations, you have just minted your first NFT!
    }
    ```
 
-2. You can view your NFT utilizing the [Terra Station NFT page](https://station.terra.money/nft). Connect your wallet using the **Connect** button located in the upper right-hand corner of the screen.
```
Comment (evanorti) [inline docs/develop/terrain/mint-an-nft.mdx:287]: @ericHgorski @terran6 This URL will not exist anymore once the new station launches.

File: docs/develop/terrain/mint-an-nft.mdx [hunk 2]
```diff
@@ -312,7 +312,7 @@ From here, you can **View** relevant information about your NFT or **Send** your
 
 ## Congratulations!
 
-You've just minted your first NFT on the Terra blockchain! If you have your own unique digital creation that you would like to share with the world, you can follow these instructions using the mainnet network and your personal Terra Station wallet.
```
Comment (evanorti) [inline docs/develop/terrain/mint-an-nft.mdx:315]: @terran6 Same here. Station will not have the NFT page.

File: docs/develop/wallet-provider/sign-bytes.mdx [hunk 3]
```diff
@@ -100,6 +100,7 @@ export function SignBytesSample() {
   );
 }
 ```
+
 You can find this code being used in context [on GitHub](https://github.com/terra-money/wallet-provider/blob/main/templates/create-react-app/src/components/SignBytesSample.tsx).
 
-You can view a working sandbox example of bytes signing with Terra Station on [codesandbox.io](https://codesandbox.io/s/github/terra-money/wallet-provider/tree/main/templates/create-react-app).
```
Comment (evanorti) [inline docs/develop/wallet-provider/sign-bytes.mdx:105]: @ericHgorski Will this sandbox still work?

File: docs/develop/wallet-provider/station-extension.mdx [hunk 4]
```diff
@@ -1,17 +1,16 @@
...
-# Terra Station extension
+The API for the Station extension gets updated periodically. If you are developing a dApp, please check regularly for updates as breaking changes may be introduced frequently.
 
-The API for the Terra Station extension gets updated periodically. If you are developing a dApp, please check regularly for updates as breaking changes may be introduced frequently.
+## What is the Station extension?
 
-## What is the Terra Station extension?
-
-The Terra Station extension is a web-wallet extension for Chrome or Firefox that enables webpages to create requests for signing and broadcasting transactions. The webpage can detect the presence of the Terra Station Extension and generate a prompt whereby the user can confirm and sign transactions.
+The Station extension is a web-wallet extension for Chrome or Firefox that enables webpages to create requests for signing and broadcasting transactions. The webpage can detect the presence of the Station Extension and generate a prompt whereby the user can confirm and sign transactions.
```
Comment (evanorti) [inline docs/develop/wallet-provider/station-extension.mdx:9]: @ericHgorski Are we going to have a firefox version of new station?

File: docs/full-node/manage-a-terra-validator/faq.mdx [hunk 5]
```diff
@@ -119,14 +119,13 @@ All Delegators have the same state as their validator.
...
 ### What is "self-bonding"? How can I increase my "self-bond"?
 
 A validator operator's "self-bond" refers to the amount of Luna delegated to itself. You can increase your self-bond by delegating more Luna to your validator account.
 
 ### Can I delegate to a validator outside of the active set?
 
-You can still delegate to a validator even if they do not appear on Terra Station. Simply add the terravaloper address of your desired validator to the end of the following URL:
+You can still delegate to a validator even if they do not appear on Station. Simply add the terravaloper address of your desired validator to the end of the following URL:
 
 `https://station.terra.money/validator/<terravaloper-address>`
```
Comment (evanorti) [inline docs/full-node/manage-a-terra-validator/faq.mdx:130]: @ericHgorski will this still work?

File: docs/learn/station/contracts.mdx [hunk 6]
```diff
@@ -1,53 +0,0 @@
-# Contract
```
Comment (evanorti) [inline docs/learn/station/contracts.mdx:1]: RIP

File: docs/learn/station/multisig.mdx [hunk 7]
```diff
@@ -2,17 +2,17 @@ import Admonition from '@theme/Admonition';
 
 # Multisig wallets
```
Comment (evanorti) [inline docs/learn/station/multisig.mdx:3]: @ericHgorski Multisig setup is unchanged, correct?

File: docs/learn/station/networks.mdx [hunk 8]
```diff
@@ -2,18 +2,18 @@ import Admonition from '@theme/Admonition';
...
 
-2. Click the settings icon in the upper right-hand corner.  In Terra Station desktop the settings icon is a globe, in the browser extensions the icon is a gear.
+2. Click the settings icon in the upper right-hand corner. In Station desktop the settings icon is a globe, in the browser extensions the icon is a gear.
 
 3. Navigate to the **Network** tab and select the network you would like to connect to.
 
 <Admonition type="tip" icon="💡" title="LocalTerra for local development">
 
-If you are interested in local development on Terra, it is recommended to utilize the LocalTerra network.  Please view the [LocalTerra documentation](../../develop/localterra/README.mdx) for more information.
+If you are interested in local development on Terra, it is recommended to use the LocalTerra network. Please view the [LocalTerra documentation](../../develop/localterra/README.mdx) for more information.
```
Comment (evanorti) [inline docs/learn/station/networks.mdx]: Can you switch to the LocalTerra network in Station?

**Review by evanorti — APPROVED** [review]

File: docs/develop/terrain/mint-an-nft.mdx (hunk 1 above)
Comment (evanorti) [reply docs/develop/terrain/mint-an-nft.mdx:287]: There should be other ways to view the NFT though, correct?

File: docs/develop/wallet-provider/sign-bytes.mdx (hunk 3 above)
Comment (ericHgorski) [reply docs/develop/wallet-provider/sign-bytes.mdx:105]: should be unaffected for the moment, although its not actually working for me as is.

File: docs/develop/wallet-provider/station-extension.mdx (hunk 4 above)
Comment (ericHgorski) [reply docs/develop/wallet-provider/station-extension.mdx:9]: yes

File: docs/full-node/manage-a-terra-validator/faq.mdx (hunk 5 above)
Comment (ericHgorski) [reply docs/full-node/manage-a-terra-validator/faq.mdx:130]: yes

File: docs/learn/station/networks.mdx (hunk 8 above)
Comment (ericHgorski) [reply docs/learn/station/networks.mdx]: not currently

**Pull request merged**